File: ball.py
class Ball:
    def __init__(self,n):
        # TODO

    def add_pair(self,a,b):
        # TODO

    def calculate(self):
        # TODO

if __name__ == "__main__":
    b = Ball(4)
    print(b.calculate()) # 0
    b.add_pair(1,2)
    print(b.calculate()) # 1
    b.add_pair(1,3)
    b.add_pair(3,2)
    print(b.calculate()) # 2

File: Java_template/Ball.java
public class Ball {
    public Ball(int n) {
        // TODO
    }

    public void addPair(int a, int b) {
        // TODO
    }

    public int calculate() {
        // TODO
    }

    public static void main(String[] args) {
        Ball b = new Ball(4);
        System.out.println(b.calculate()); // 0
        b.addPair(1,2);
        System.out.println(b.calculate()); // 1
        b.addPair(1,3);
        b.addPair(3,2);
        System.out.println(b.calculate()); // 2
    }
}
